docs/reference-architectures/sap/index.md

```diff
@@ -0,0 +1,418 @@
+
+\* For details about SMLG and Azure Load Balancer, see the “Load balancer” item
+in the following “Architecture” section.
+
+> [!NOTE]
+-   SAP HANA is certified for production OLAP workloads on Azure GS5-series
+    virtual machines. This reference architecture is for Azure virtual machines
+    in the G-series and M-series and differs from [SAP HANA on Azure Large Instances][azure-large-instances].
+
+-   Deployment of this reference architecture requires appropriate licensing of
+    SAP products and other non-Microsoft technologies.
+
+-   For information about the partnership between Microsoft and SAP, see [SAP HANA on Azure][sap-hana-on-azure]. 
+>
+
+## Architecture
+
+This reference architecture depicts a distributed installation of SAP applications. There are several databases that SAP applications can use, and here the SAP HANA database is described. Although this document focuses on a highly available environment within a single Azure region, all components of this architecture can be replicated to secondary regions for disaster recovery purposes.
+
+The architecture consists of the following components.
+
+-   **Virtual network**. The Azure Virtual Network service securely connects
+    Azure resources to each other. A virtual network is a representation of your
+    own network in the cloud—that is, a logical isolation of the Azure cloud
+    dedicated to your subscription. You can also connect virtual networks to
+    your network on premises.
+
+-   **Subnets**. A subnet defines a logical collections of network resources by
+    specifying address ranges within a virtual network. Subnets make it easier
+    to manage and segregate network traffic without impacting performance. For
+    example, you can create a separate subnet for each tier: application (SAP
+    NetWeaver), database (SAP HANA), and management (the jumpbox).
+
+-   **Availability sets**. These are groups of virtual machines performing the
+    same role. Using availability sets improves the availability of a service
+    during system failure or routine cloud platform maintenance.
+
+-	**Load balancers.** The SAP Web Dispatcher component is used as a load balancer 
+    for SAP traffic among the SAP application servers. To achieve high availability,
+    the [Azure Load Balancer][azure-lb] (shown on the left in Figure 1) is used
+    to implement the failover cluster for the SAP web dispatchers. 
+    SMLG, the internal load balancer in ABAP
+    Central Services (ASCS), is used for balancing the SAP application server
+    pool for SAPGUIs and RFC traffic.  An Azure [internal load balancer][ilb]
+     (shown on the right in Figure 1) is also used to implement the ASCS Windows
+    Server Failover Cluster (WSFC). The application server connection to the
+    highly available ASCS is through the cluster virtual network name. Assigning
+    the cluster virtual network name to the endpoint of this internal load
```

Comment:
Speaking of cluster virtual network name ... does this RA need a domain controller for the WSFC?